### ISSUE TITLE
Poker has severe issues

### ISSUE THREAD

**Issue by Rammko** [SaitoTech/saito-lite-rust#331]
When a player raises, the opponent is randomly forced to call a different higher number with decimals that either FORCE the opponent to fold, or end the game outright
![image](https://user-images.githubusercontent.com/61713323/160900396-202b6198-2913-4046-a0fa-d6985efb3ceb.png)
![image](https://user-images.githubusercontent.com/61713323/160900563-7bdd66f1-5e91-4e1f-97a0-42b97b916cc2.png)

There are also some cases where when trying to raise, the number to raise is "NaN" which end the game immediately as "Undefined Wins"
Additionally, when a player folds on the initial hand, there is a loss of overall chips

In short, Poker is unplayable in its current state, please revert to previous version
